doc/README.md

```diff
@@ -42,59 +73,163 @@ services:
+2. Install necessary requirements for development:
 
-Selenium tests are designed to run against a certain database state and service ports.
+```
+pip3 install --upgrade pip setuptools wheel
+pip3 install -r requirements.txt
+```
+
+3. Create a `local_settings.py` -file at `a-plus/local_settings.py`.
+There is an example file `local_settings.example.py`, which you can use as a starting point.
+
+4. Ensure your `local_settings.py` contains at least the following lines:
+
+```
+DEBUG = True
+BASE_URL = 'http://localhost:8000/'
+```
+
+5. Run the existing django migrations:
+
+```
+python manage.py migrate
+```
+
+6. You can now run the local version without a container:
+
+```
+python manage.py runserver
+```
+
+If you need to be able to login or use the admin-page while running the code without a container, you can create a superuser by following [these instructions](https://docs.djangoproject.com/en/3.0/intro/tutorial02/#introducing-the-django-admin). 
+
+### Unit tests
+
+1. Run the unit tests:
+
+```
+python manage.py test
+```
+
+### Selenium integration tests
+
+Selenium tests check end-to-end -functioning from a user-perspective.
 The tests will automatically open and direct Firefox browser windows.
-Currently, the tests depend on Unix type shell.
-All the following commands assume that the project virtualenv has been activated:
+Currently, the tests depend on a Unix type shell.
+
+1. Install the necessary requirements:
+
+```
+pip install -r requirements_testing.txt
+```
+
+2. To setup the servers and run all the tests at one go:
+
+```
+selenium_test/run_servers_and_tests.sh
+```
 
-		cd [project_root]
-		source PATH_TO_VIRTUALENV/bin/activate
+3. Alternatively you can run individual tests:
 
-### Prerequisites
+```
+cd selenium_test/test/
+../run_servers.sh
+
+python login_test.py
+python home_page_test.py
+
+../kill_servers.sh
+```
+
+### Updating translations for Finnish and English versions
 
-  - Firefox browser installed
-  - Python modules for testing in the virtualenv (`pip install -r requirements_testing.txt`)
+Before creating a pull request, you need to ensure that [translations](https://github.com/apluslms/a-plus/blob/master/locale/fi/LC_MESSAGES/django.po) are up-to-date.
+Translations are handled with [Django](https://docs.djangoproject.com/en/3.1/ref/django-admin/#django-admin-makemessages) and are done outside the container.
 
-### Running
+1. In order to update the file containing translations, run:
 
-To setup the servers and run all the tests at one go:
+```
+python manage.py makemessages --no-obsolete
+```
 
-	selenium_test/run_servers_and_tests.sh
+2. You can easily check the lines that were affected:
 
-Running individual tests:
+```
+git diff locale/fi/LC_MESSAGES/django.po
+```
 
-	cd selenium_test/test/
-	../run_servers.sh
+3. Add new Finnish translations for the lines you have created.
 
-	python login_test.py
-	python home_page_test.py
+4. And compile the translations to ensure they are working:
```

Comment:
This step also makes translations work when code is mounted to `/srv/aplus`.

Reply:
I added a mention of that